simulator_link

```diff
@@ -0,0 +1 @@
+bin/simulator
```

Comment:
I forget why I needed to do this but I remember it was a really gross hack for something and we probably shouldn't get used to doing things this way. I'll see if I can figure it out soon.

Reply:
I have no idea why this was necessary. Did you figure out why? I'd be more than happy to remove it completely.